samples/Microsoft.Azure.CosmosEventSourcing/BasicEventSourcingSample/Program.cs

```diff
@@ -13,13 +13,12 @@
 using Microsoft.Azure.CosmosEventSourcing.Extensions;
 using Microsoft.Azure.CosmosEventSourcing.Stores;
 using Microsoft.Azure.CosmosRepository.AspNetCore.Extensions;
-using Microsoft.Azure.CosmosRepository.Extensions;
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 IServiceCollection services = builder.Services;
 
-
-services.AddHealthChecks().AddCosmosDb(builder.Configuration.GetCosmosRepositoryConnectionString()!);
+// TODO: Figure this out...
+// services.AddHealthChecks().AddCosmosDb(builder.Configuration.GetCosmosRepositoryConnectionString()!);
```

Comment:
Has this broken?

Reply:
Yeah, with the latest version the `AddHealthChecks` method doesn't exist, and I didn't have time to look it up. It was in a sample app so I left it for now, but happy to hold this for a bit.

Reply:
Fixed it...